cli/cmd/profile.go

```diff
@@ -118,11 +147,121 @@ Example:
 
 	cmd.PersistentFlags().BoolVar(&options.template, "template", options.template, "Output a service profile template")
 	cmd.PersistentFlags().StringVar(&options.openAPI, "open-api", options.openAPI, "Output a service profile based on the given OpenAPI spec file")
+	cmd.PersistentFlags().StringVar(&options.tap, "tap", options.tap, "Output a service profile based on tap data for the given target resource")
```

Comment:
What do you think about a flag that limits the maximum number of routes to output?  By setting a default of something like 100, we can safeguard against out-of-control profiles for high traffic REST-y services.

Reply:
yeah that's a good idea!